performance-tracking/src/main/java/com/rakuten/tech/mobile/perf/runtime/internal/AnalyticsBroadcaster.java

```diff
@@ -69,4 +83,24 @@ public void sendEvent(@NonNull String name, @Nullable Map<String, ?> data) {
     Context ctx = this.context.get();
     if (ctx != null) AnalyticsBroadcaster.sendEvent(ctx, name, data);
   }
+
+  /**
+   * Checks if an event is recording an HTTP request to a blacklisted domain.
+   *
+   * @param url String url of the recorded HTTP request.
+   * @return true if event records an HTTP request to a blacklisted domain, false otherwise.
+   */
+  @Override
+  protected boolean isUrlBlacklisted(String url) {
+    try {
+      URL candidate = new URL(url);
+      for (URL blacklisted : blacklist) {
+        if (blacklisted.getHost().equals(candidate.getHost())) {
```

Comment:
This implementation will ignore the path if something like `https://www.example.com/endpoint` is added to the blacklist. Do we need to support this use case?

Reply:
hmm we spoke about blacklisting by domain only, but that's a valid concernt. it could also be a 
`candidate.getPath().startsWith(blacklisted.getPath()` OR'red into the conditional. Let's discuss with vito and donnie

Reply:
i'll merge for now to start feature QA